Resources/doc/config.rst

```diff
@@ -373,7 +373,7 @@ string as a comma separated list.
             # ...
             connections:
                 default:
-                    server: "mongodb://mongodb-01:27017,mongodb-02:27017,mongodb-03:27017"
+                    server: "mongodb://mongodb-01:27017,mongodb-02:27017,mongodb-03:27017/?replicaSet=replSetName"
```

Comment:
@alcaeus is this necessary?

Reply:
Yes, if you are connecting to a replica set. See the corresponding [documentation](https://docs.mongodb.com/manual/reference/connection-string/#replica-set-option).